_partials/_migrate_live_run_live_migration.md

```diff
@@ -0,0 +1,88 @@
+1. **Pull the live-migration docker image to you migration machine**
+
+   ```shell
+   sudo docker pull timescale/live-migration
```

Comment:
Added sudo because I could not pull the image without running as Sudo on my default ec2 image. 

When I reviewed the doc, the release tag was 3 versions out. I'm sure it is best practice to specify the version, but unless we are sure to update this version number in the docs, I figured it best to go for an implicit latest. Happy to go for version number if that is best practice.